src/test/java/com/vinylteam/vinyl/dao/jdbc/JdbcShopDaoITest.java

```diff
@@ -91,12 +103,23 @@ void getManyByListOfIdsFromEmptyTable() throws SQLException {
         assertTrue(actualShops.isEmpty());
     }
 
+    @Test
+    @DisplayName("Gets empty list of all shops from empty table")
+    void getAllShopsFromEmptyTable() throws SQLException {
```

Comment:
move it next to getAllShops() - it's easier to see all tests for one method together.

findAll().